manifests/charts/istio-cni/templates/configmap-cni.yaml

```
@@ -14,13 +14,13 @@ data:
   AMBIENT_ENABLED: {{ .Values.ambient.enabled | quote }}
   AMBIENT_DNS_CAPTURE: {{ .Values.ambient.dnsCapture | default "false" | quote  }}
   AMBIENT_IPV6: {{ .Values.ambient.ipv6 | default "false" | quote }}
-  {{- if .ValuesConfFileName }} # K8S < 1.24 doesn't like empty values
-  CNI_CONF_NAME: {{ .ValuesConfFileName }} # Name of the CNI config file to create. Only override if you know the exact path your CNI requires..
+  {{- if .Values.global.cniConfFileName }} # K8S < 1.24 doesn't like empty values
+  CNI_CONF_NAME: {{ .Values.global.cniConfFileName }} # Name of the CNI config file to create. Only override if you know the exact path your CNI requires..
```

```suggestion
  CNI_CONF_NAME: {{ .Values.cniConfFileName }} # Name of the CNI config file to create. Only override if you know the exact path your CNI requires..
```

It should not be under global?

---

(same to every other part of the PR)

---

Ohh, my mistake. Let me check why I did not change that. I also asked you those questions in the issue linked